components/wmtk_components/embedded_component/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+
+set(SRC_FILES
+    internal/EmbeddedRemeshingOptions.hpp
+    internal/EmbeddedRemeshing.hpp
+    internal/EmbeddedRemeshing.cpp
+	embedded_remeshing.hpp
```

Comment:
Fix indentation. We don't have a style guide for CMake files so just try to follow the style you can find in existing CMake files.